website/content/docs/synced-folders/nfs.mdx

```diff
@@ -164,8 +164,8 @@ Cmnd_Alias VAGRANT_EXPORTS_REMOVE = /usr/bin/sed -E -e /*/ d -ibak /etc/exports
 For Ubuntu Linux , sudoers should look like this:
 
 ```
-Cmnd_Alias VAGRANT_EXPORTS_CHOWN = /bin/chown 0\:0 /tmp/*
-Cmnd_Alias VAGRANT_EXPORTS_MV = /bin/mv -f /tmp/* /etc/exports
+Cmnd_Alias VAGRANT_EXPORTS_CHOWN = /bin/chown 0\:0 /tmp/vagrant[a-z0-9-]*
+Cmnd_Alias VAGRANT_EXPORTS_MV = /bin/mv -f /tmp/vagrant[a-z0-9-]* /etc/exports
```

Comment:
Thanks for diving into this! A couple of questions here:

1. Would it be possible to bypass this restriction with a slightly more complicated path traversal, i.e. : `/tmp/vagrant1/../../../home/user/sudo_exploit`?
2. Secondly, could you do the same thing with a symlink?
```
ln -s /home/user/sudo_exploit /tmp/vagrant1
# Continue exploit here
```

Reply:
No, it will not work. With the matching set enforced you are not able to include extra arguments which was required to relocate the file.